metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -249,6 +249,11 @@ def __init__(self, ctx: PipelineContext, config: BigQueryV2Config):
             config, self.report, self.profiling_state_handler
         )
 
+        # Global store of table identifiers for lineage filtering
+        self.table_refs: Set[str] = set()
```

Comment:
I think we should support ingestion recipes which only ingests lineage metadata and not even try to get schema metadata.
I can see use-cases where there is a project which is only for submitting queries/jobs and they don't want to ingest metadata from that project at all. As all the queries are submitted to that project we have to process the gcp logs to get lineage/usage. Or maybe we need a property to be able to set a project which we use to get lineage and if nothing is set we try to get it from all the project we discover?

Reply:
Totally agree. Although, that is not scope of this PR. We should definitely discuss this and implement that feature.

Reply:
WDYT @treff7es ?